pkg/model/loki.go

```diff
@@ -20,6 +20,7 @@ type AggregatedQueryResponse struct {
 	ResultType ResultType      `json:"resultType"`
 	Result     ResultValue     `json:"result"`
 	Stats      AggregatedStats `json:"stats"`
+	IsMock     bool            `json:"isMock"`
```

Comment:
I wonder if there is a better way than exposing whether the response is mocked 🤔

Reply:
Yes, I wanted to pass this from `/api/frontend-config` route however I found more issues than anything since the `/api/loki/topology` one is called before getting the config.